src/main/java/seedu/duke/commands/Command.java

```diff
@@ -18,5 +19,5 @@
      * @throws ClickException If there is an exception of type ClickException occurs.
      * @throws Exception If there is an exception of type other than DukeException occurs.
      */
-    public abstract void execute(Ui ui, Storage storage) throws ClickException, Exception;
+    public abstract void execute(TaskList taskList, Ui ui, Storage storage) throws ClickException, Exception;
```

Comment:
Hmm, would it be better if the `taskList` is found within `storage` instead, reason being I was thinking of how to implement other listing features too - we need to add another parameter for every additional adding feature, may be a very long method call, wdyt?

For example, `whatIAteTodayList` is found in `storage` object

Reply:
sure